### DISCUSSION TITLE
[DatePicker] 强化时间选择

### DISCUSSION THREAD

**Discussion by xmsz-stu** [DouyinFE/semi-design#2296] [Announcements]
### Which Component Need Enhancement 期望新增功能的组件
- DatePicker

### What does the proposed API look like 期望支持的API
强化时间选择

### Feature Description 功能描述
现在是强化日期，有些场景下是需要强化时间选择或者只是两者平等

现在的交互是，先选日期，然后用户自己得手动点击右下角时间，再选择时间
一些用户反馈不会用，不习惯。

我觉得可以有几点处理

第一个，点击日期后自动调整到时间选择器，这样用户不会忘记选时间，而且操作连贯，缺点就是帮用户操作了
第二个，左右两边布局，这个是最常见的（毕竟原生就是这么展示），也是用户最能接受的习惯
![image](https://github.com/DouyinFE/semi-design/assets/139225486/f7fee76c-b08f-4730-9655-3421e4198a73)
![image](https://github.com/DouyinFE/semi-design/assets/139225486/73595ac7-e9b8-4f2d-9faf-26f41aa652e4)
![image](https://github.com/DouyinFE/semi-design/assets/139225486/c1f3547d-464b-4ffd-8d47-5a33f9dd6847)

第三个，上下布局，顺序问题，区别不大
![image](https://github.com/DouyinFE/semi-design/assets/139225486/0199f2d8-4b61-424a-a5fa-cbf425b3048f)
![image](https://github.com/DouyinFE/semi-design/assets/139225486/09db27c6-7943-4ae3-b989-713d3c4a602f)

### Additional information 补充说明
- 如果拆成两个组件分开选择日期和时间，用户操作更麻烦
- 如果能单独提供面板也行，我们自己拼在一起

**Comment by shijiatongxue** [comment]
这个想法挺好的。不过这个ui实现跟我们的ui实现区别有点大。我们短时间内不太可能会太大的变化，这个我们后面升级大版本的时候可以考虑进来。

**Comment by shijiatongxue** [comment]
我准备把这个 issue 转为 discussion 了，大家可以继续到里面讨论
